src/worker/worker-options.ts

```diff
@@ -1,11 +1,24 @@
+import type { KillBehavior } from '../utility-types'
+
+/**
+ * Kill behavior enumeration
+ */
+export const killBehaviorEnumeration = Object.freeze({
```

Comment:
I think you can just name killBehavior. The type of data structure is not necessary to be put in the variable name.

Reply:
Yes I think killBehaviourTypes would be a better name

Reply:
Done, thanks jerome